### PR TITLE
Release v0.28.0

### PR DESCRIPTION
**Pull request by eunicorn**
bump version to v0.28.0 and generate change log

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,5 +1,12 @@
 # Changelog
 
+## 0.28.0 / 2023-04-17
+
+- [FEATURE] Revamp project page (#1062)
+- [ENHANCEMENT] Highlight behavior slight improvement using opacity blur (#1107)
+- [ENHANCEMENT] Increase tooltip max height to fit more series w/o scrolling (#1111)
+- [BREAKINGCHANGE] Define queries in panel spec instead of panel plugin spec (#1032)
```
Comment (eunicorn) [inline CHANGELOG.md:8]: let me know if i should include https://github.com/perses/perses/pull/1060 even though it was merged into #1032

Comment (juliepagano) [inline CHANGELOG.md:8]: I'm not sure if an outside observer would know exactly how to handle this breaking change. I'm not sure if some extra notes should go in here or in the description on the linked PR. Leaning towards the latter, but I'm curious what other folks think.

Comment (eunicorn) [reply CHANGELOG.md:8]: I think adding more details in the linked PR should be ok?

**Review by sjcobb — APPROVED** [review]

**Review by juliepagano — APPROVED** [review]
🚀🚀🚀

**Review by AntoineThebaud — APPROVED** [review]

**Pull request merged**